### PR TITLE
Conditional sidebar menu in statement detail

### PR DESCRIPTION
**Pull request by jalalifardemos**
### Ticket: https://demoseurope.youtrack.cloud/issue/DPLAN-13002

 Description: use permission to show sidebar menu  

Delete the checkbox if it doesn't apply/isn't necessary. 

- [ ] Create/Update tests
- [ ] Update documentation
- [ ] Add/Update data-cy attributes ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))
- [ ] Run `yarn lint`
- [ ] Run `yarn test`
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue [hunk 1]
```diff
@@ -232,8 +232,8 @@ export default {
       menuEntries: [
         { id: 'entry', transKey: 'entry' },
         { id: 'submitter', transKey: 'submitted.author' },
-        { id: 'publicationAndVoting', transKey: 'publication.and.voting' },
-        { id: 'locationAndDocuments', transKey: 'location.and.document.reference' },
+        { id: 'publicationAndVoting', transKey: 'publication.and.voting', condition: hasPermission('feature_statements_vote') || hasPermission('feature_statements_publication') },
```
Comment (salisdemos) [inline client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue:235]:
```suggestion
        { id: 'publicationAndVoting', transKey: 'publication.and.voting', condition: hasAnyPermissions(['feature_statements_vote', 'feature_statements_publication']) },
```
but then You have to add `hasAnyPermissions` in the import section in he demosplan-ui block

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue (hunk 1 above)
Comment (jalalifardemos) [reply client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue]: done in https://github.com/demos-europe/demosplan-core/pull/4130/commits/cad05829d871fdd55860c7a93731f3bc7bacfa58

Comment (salisdemos) [reply client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue]: sorry, but you did not read the comment below my suggestion

Comment (jalalifardemos) [reply client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue]: Sorry, done in https://github.com/demos-europe/demosplan-core/pull/4130/commits/4098c6c297bf40d01f3c0612895ba93b8d103a41

File: client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue [hunk 2]
```diff
@@ -126,6 +126,7 @@ import {
   DpLabel,
   DpSelect,
   DpTextArea,
+  hasAnyPermissions,
```
Comment (salisdemos) [inline client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue:129]: please put below `dpValidateMixin` to order alphabetically

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue (hunk 2 above)
Comment (jalalifardemos) [reply client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue]: done in https://github.com/demos-europe/demosplan-core/pull/4130/commits/c5c4f42730068abb4536abcc58b2a960130effb2

File: client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue [hunk 3]
```diff
@@ -254,6 +255,10 @@ export default {
       return today
     },
 
+    filterMenue () {
```
Comment (salisdemos) [inline client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue:258]: sorry, that I didn't see it before:
This is a method, but its more like an dynamic variable, so the wording should be `filteredMenuEntries`

Welcome to the world of vue ;-)

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue (hunk 3 above)
Comment (jalalifardemos) [reply client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue]: like alice in wonderland :)) https://github.com/demos-europe/demosplan-core/pull/4130/commits/b207b6293b2842890d0625246234926e7ee9a5e2

**Review by salisdemos — APPROVED** [review]
welcome to the Frontend.

**Pull request merged**